### PR TITLE
Modal window

### PR DESCRIPTION
**Pull request by olenakorol1708**
@lluneva 

Added a modal window after form submission confirmation. When the modal window is closed, the user will be redirected to the home page.

### CODE REVIEW

File: src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx [hunk 1]
```diff
@@ -1,11 +1,50 @@
...
+  
+    const handleOk = () => {
+        handleModalClose(); 
+      navigate('/');
+    };
+  
+    return (
+      <Modal
+        title={null}
+        visible={isModalVisible}
```
Comment (lluneva) [inline src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx:23]: image

File: src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx [hunk 2]
```diff
@@ -1,11 +1,50 @@
...
+          <img
+            src={smile}
+            alt="smile"
+            style={{ width: '50px', marginBottom: '10px' }}
+          />
+          <h3>Thank you!</h3>
+          <p>We have received your message and will be in contact shortly!</p>
+          <Button
+            type="primary"
+            style={{ width: '100%', backgroundColor: '#c3f00f', marginTop: '2rem', color: '#121c22' }}
```
Comment (lluneva) [inline src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx:39]: using inline styles are not the best  practice in general. https://www.thoughtco.com/avoid-inline-styles-for-css-3466846

File: src/Pages/RegisterForm/style.scss [hunk 3]
```diff
@@ -38,7 +44,8 @@
       border: 1px solid #d1d0d0;
       font-size: 16px;
       font-family: $main-font;
-      color: #243843 !important;
+      // color: #243843 !important;
```
Comment (lluneva) [inline src/Pages/RegisterForm/style.scss]: commented line sis also bad practice

File: src/Pages/RegisterForm/RegisterForm.tsx [hunk 4]
```diff
@@ -91,6 +93,8 @@ const App: React.FC = () => {
     
 
       form.resetFields();
+      setIsModalVisible(true);
```
Comment (lluneva) [inline src/Pages/RegisterForm/RegisterForm.tsx:92]: I assume you're not making an actual backend call, right? If that's the case, you could simulate it by using a setTimeout function that waits for 3 seconds. You can randomly resolve it with a 50% chance of success, where the modal you created is shown, and a 50% chance of failure, where a different message like "Something went wrong, please try again later" is displayed.

By doing this, you'd incorporate a few key aspects of front-end development:

1. Handling a loading state: You can show a loading indicator while waiting for the simulated response, ensuring a smooth user experience during longer operations.
2.  Managing result handling: You'll need to handle both success and failure cases properly. On success, show a confirmation message in the modal. On failure, display an error message and potentially allow the user to retry the operation.

File: src/Pages/RegisterForm/style.scss (hunk 3 above)
Comment (olenakorol1708) [reply src/Pages/RegisterForm/style.scss]: fixed

File: src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx (hunk 1 above)
Comment (olenakorol1708) [reply src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx:23]: changed prop visible to 'open'

File: src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx (hunk 2 above)
Comment (olenakorol1708) [reply src/Pages/RegisterForm/ModalWindow/ModalWindow.tsx:39]: fixed. Added className = 'modal-button'

File: src/Pages/RegisterForm/RegisterForm.tsx (hunk 4 above)
Comment (olenakorol1708) [reply src/Pages/RegisterForm/RegisterForm.tsx:92]: Added

**Pull request merged**